web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -30,7 +30,11 @@
 public class WebUtil implements GlobalPropertyListener {
 	
 	private static final Logger log = LoggerFactory.getLogger(WebUtil.class);
-	
+
+	/**
+	 * @deprecated as of 2.2.0-SNAPSHOT, there is no replacement field available.
```

Comment:
That should be the version to be released. And hence it is 2.2.0

Reply:
I will get this fixed

Reply:
Should be fixed now